src/wallet/partialTxProposal.ts

```diff
@@ -118,16 +129,32 @@ class PartialTxProposal {
 
   /**
    * Add an UTXO as input on the partial data.
-   * This need to be async since we may need to fetch the transaction.
    *
    * @param {string} hash Transaction hash
    * @param {number} index UTXO index on the outputs of the transaction.
-   * @param {Object} [options]
+   * @param {number} value UTXO value.
+   * @param {Object} [options={}]
```

Comment:
I agree that is more readable, but the actual default value is `{}` and each property has a default when destructuring.
But since this is more confusing i'll add your suggestion here and in some other places with this.